src/convert.spec.ts

```diff
@@ -67,57 +69,62 @@ describe('convert()', () => {
 					},
 				},
 			}),
-		).toMatchObject({
-			'3303:1.1': [
-				{
-					'5518': 1665149602,
-					'5601': 23.51,
-					'5602': 23.51,
-					'5603': -40,
-					'5604': 85,
-					'5700': 24.57,
-					'5701': 'Celsius degrees',
-				},
-			],
-			'3304:1.1': [
-				{
-					'5518': 1665149602,
-					'5601': 31.064,
-					'5602': 31.064,
-					'5603': 0,
-					'5604': 100,
-					'5700': 28.927,
-					'5701': '%',
+			{
+				'3303:1.1': [
+					{
+						'5518': 1665149602,
+						'5601': 23.51,
+						'5602': 23.51,
+						'5603': -40,
+						'5604': 85,
+						'5700': 24.57,
+						'5701': 'Celsius degrees',
+						'5750': '',
+					},
+				],
+				'3304:1.1': [
+					{
+						'5518': 1665149602,
+						'5601': 31.064,
+						'5602': 31.064,
+						'5603': 0,
+						'5604': 100,
+						'5700': 28.927,
+						'5701': '%',
+						'5750': '',
+					},
+				],
+				'50009@1.2': {
+					'1': 10.0,
+					'2': 5.0,
+					'3': 60.0,
+					'4': 120,
+					'5': true,
+					'6': 30,
+					'7': 120,
+					'8': 3600,
+					'9': true,
+					'10': true,
 				},
-			],
-			'50009@1.2': {
-				'1': 10.0,
-				'2': 5.0,
-				'3': 60.0,
-				'4': 120,
-				'5': true,
-				'6': 30,
-				'7': 120,
-				'8': 3600,
-				'9': true,
-				'10': true,
 			},
-		}))
+		)
+	})
 
-	it('should throw an exception if the input cannot be converted', () =>
-		expect(() =>
+	void it('should throw an exception if the input cannot be converted', () => {
+		const error = () =>
 			convert({
 				foo: {
 					'0': {
 						'Application Type': '',
 					},
 				},
-			}),
-		).toThrow())
+			})
+		assert.throws(error)
```

Comment:
```suggestion
		assert.throws(() =>
			convert({
				foo: {
					'0': {
						'Application Type': '',
					},
				},
			}))
```

This removes the not so helpful naming of the variable.

Reply:
https://github.com/NordicSemiconductor/asset-tracker-cloud-coiote-aws-converter-js/pull/85/commits/7ca94bee1cb6159a7ae9165f3bba09d5c204f0e0